docs/source/user/api.rst

```diff
@@ -378,6 +378,20 @@ stop_time           Stop time (less or equal to)
 is_open             If stop_time is null
 ==================  ====================================
 
+Signals
++++++++
+
+``radius_accounting_success``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Path**: ``openwisp_radius.signals.radius_accounting_success``
+
+**Arguments**:
+
+- ``accounting_data``: accounting information
+
+This signal is emitted every time accounting information is added or updated.
+
```

Comment:
Please let's add a new page as requested in the [issue description](https://github.com/openwisp/openwisp-radius/issues/341) because this will likely grow in the near future.